coreax/kernel.py

```diff
@@ -48,35 +48,34 @@
 
 from typing_extensions import deprecated
 
-from coreax.kernels.base import (
-    AdditiveKernel,
-    DuoCompositeKernel,
-    ProductKernel,
-    ScalarValuedKernel,
-    UniCompositeKernel,
+from coreax.kernels.base import AdditiveKernel as _AdditiveKernel
+from coreax.kernels.base import DuoCompositeKernel as _DuoCompositeKernel
+from coreax.kernels.base import ProductKernel as _ProductKernel
+from coreax.kernels.base import ScalarValuedKernel as _ScalarValuedKernel
+from coreax.kernels.base import UniCompositeKernel as _UniCompositeKernel
+from coreax.kernels.scalar_valued import ExponentialKernel as _ExponentialKernel
+from coreax.kernels.scalar_valued import LaplacianKernel as _LaplacianKernel
+from coreax.kernels.scalar_valued import LinearKernel as _LinearKernel
+from coreax.kernels.scalar_valued import LocallyPeriodicKernel as _LocallyPeriodicKernel
+from coreax.kernels.scalar_valued import PCIMQKernel as _PCIMQKernel
+from coreax.kernels.scalar_valued import PeriodicKernel as _PeriodicKernel
+from coreax.kernels.scalar_valued import PolynomialKernel as _PolynomialKernel
+from coreax.kernels.scalar_valued import (
+    RationalQuadraticKernel as _RationalQuadraticKernel,
```

Comment:
These can be done as  just two statements (like before), with 
```python
from coreax.kerenels.base import (
    AdditiveKernel as _AdditiveKernel,
    DuoCompositeKernel as _DuoCompositeKernel,
    ...
)
```

Reply:
I did that and it has been auto-formatted as you see it in the PR

Reply:
Weird! This can be fixed by changing a Ruff setting (`lint.isort.combine-as-imports = true`), but this actually makes some changes in other files, and also allows that (quite bad to my eyes) formatting `from jax import Array, numpy as jnp, random as jr`. To fix this latter point, also set `lint.isort.force-wrap-aliases = true`. 

So, to summarise, add the following lines to `pyproject.toml`:
```toml
[tool.ruff.lint.isort]
combine-as-imports = true
force-wrap-aliases = true
```

Alternatively, this could be a separate PR. No strong opinion on that from me.

Reply:
Added this